OpenRA.Game/Map/MapPreview.cs

```diff
@@ -226,7 +226,7 @@ public Ruleset LoadRuleset()
 		{
 			return Ruleset.Load(modData, this, TileSet, innerData.RuleDefinitions,
 				innerData.WeaponDefinitions, innerData.VoiceDefinitions, innerData.NotificationDefinitions,
-				innerData.MusicDefinitions, innerData.SequenceDefinitions, innerData.ModelSequenceDefinitions);
+				innerData.MusicDefinitions, innerData.ModelSequenceDefinitions);
```

Comment:
I'm not sure I'm comfortable with how special sprite sequences are becoming by being the only exception here.

Reply:
Sprite sequences (and terrain info) were already an exception (sometimes null, sometimes not, parsed in a completely different way)! The changes here are cleaning up those hacks.